src/storage/backend/sqlite/SQLite.cc

```diff
@@ -496,7 +496,7 @@ void SQLite::DoExpire(double current_network_time) {
     // Automatically rollback the transaction when this object is deleted.
     auto deferred_rollback = util::Deferred([this]() {
         char* errMsg = nullptr;
-        sqlite3_exec(expire_db, "rollback transaction", nullptr, nullptr, &errMsg);
+        [[maybe_unused]] int ret = sqlite3_exec(expire_db, "rollback transaction", nullptr, nullptr, &errMsg);
```

Comment:
Hmm, elsewhere there's: `sqlite3_exec(expire_db, "commit transaction", nullptr, nullptr, &errMsg);`

Maybe just ignore the return value  here, too? But would think that we should actually handle and check the result and log a message if something is off.